server/glassfish-demo/xebialabs/xlr-pipeline.yaml.tmpl

```diff
@@ -0,0 +1,96 @@
+{{$app := .AppName | kebabcase}}
+---
+apiVersion: xl-release/v1
+kind: Templates
+spec:
+- directory: {{.FolderName}}
+  children:
+  - template: {{.AppName}}-GlassFish-JIRA
+    phases:
+    - phase: Dev
+      tasks:
+      - name: Open Change Request
+        type: jira.CreateIssue
+        jiraServer: XebiaLabs JIRA
+        project: SAN
+        ^title: Sample Project Ticket
+        ^description: This is for {{.AppName}} project
+        variableMapping:
+          pythonScript.issueId: ${ticket}
+      - name: Deploy to DEV
+        type: xldeploy.Deploy
+        server: XebiaLabs Internal
+        deploymentPackage: glassfish/sample/1.0.0
+        deploymentEnvironment: glassfish/DEVglassfish
+      - name: Add comment to change request
+        type: jira.UpdateIssue
+        jiraServer: XebiaLabs JIRA
+        issueId: ${ticket}
+        newStatus: IN PROGRESS
+        ^comment: Deployed application {{.AppName}} deplyed to environment DEV
+      color: '#68b749'
+    - phase: Test
+      tasks:
+      - name: Verify application in TEST
+        type: xlrelease.Task
+        description: Verify manually at [http://localhost:9980/sample/](http://localhost:9980/sample/)
+        owner: admin
+      - name: Perform Automated Testing
+        type: xlrelease.ParallelGroup
+        tasks:
+        - name: Smoke Testing
+          type: xlrelease.ScriptTask
+          owner: admin
+          script: |-
+            import time
+            time.sleep(5)
+            print "Performing Smoke Tests"
```

Comment:
I understand that the test is not the point but since this will pass always suggestion was to add a comment in YAML to make it explicit, I'll leave it upto you to decide